features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/ViewUtilsInternalTest.kt

```diff
@@ -6,25 +6,35 @@
 
 package com.datadog.android.sessionreplay.internal.recorder
 
+import android.graphics.drawable.Drawable
 import android.view.View
 import android.view.ViewStub
+import android.widget.TextView
 import androidx.appcompat.widget.ActionBarContextView
 import androidx.appcompat.widget.Toolbar
+import com.datadog.android.api.InternalLogger
 import com.datadog.android.sessionreplay.forge.ForgeConfigurator
+import com.datadog.android.sessionreplay.internal.recorder.ViewUtilsInternal.Companion.COMPOUND_DRAWABLE_INDEX_OUT_OF_BOUNDS
```

Comment:
it should be possible to reference it directly, without `Companion`

```suggestion
import com.datadog.android.sessionreplay.internal.recorder.ViewUtilsInternal.COMPOUND_DRAWABLE_INDEX_OUT_OF_BOUNDS
```